zproject_java_lib.gsl

```diff
@@ -197,6 +224,8 @@ function resolve_method (method)
                 jni_native_invocation_c += "$(comma)($(argument.c_type)) (intptr_t) &$(argument.c_name)"
             elsif jni_is_class = 1 | type = "anything" | type = "sockish"
                 jni_native_invocation_c += "$(comma)($(argument.c_type)) (intptr_t) $(argument.c_name)"
+            elsif argument.callback ?= 1
+                jni_native_invocation_c += "$(comma) $(argument.c_name)_$(argument.type)"
```

Comment:
Big gaping hole: need to replace the "args" parameter with a pointer to a jni_cb_struct.
Which means we need to malloc that. And make sure it gets freed.